client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveHelpStatisticsTable.tsx

```diff
@@ -0,0 +1,340 @@
+        sortProps: {
+          sort: (a, b): number => {
+            return sortNullableLiveFeedbackCount(
+              a.liveFeedbackCount?.[index] ?? null,
+              b.liveFeedbackCount?.[index] ?? null,
+            );
+          },
+        },
+      };
+    });
+
+  const columns: ColumnTemplate<AssessmentLiveFeedbackStatistics>[] = [
+    {
+      searchProps: {
+        getValue: (datum) => datum.courseUser.name,
+      },
+      title: t(translations.name),
+      sortable: true,
+      searchable: true,
+      cell: (datum) => (
+        <div className="flex grow items-center">
+          <Link to={`/courses/${courseId}/users/${datum.courseUser.id}`}>
+            {datum.courseUser.name}
+          </Link>
+          {datum.courseUser.isPhantom && (
+            <GhostIcon className="ml-2" fontSize="small" />
+          )}
+        </div>
+      ),
+      csvDownloadable: true,
+    },
+    {
+      of: 'groups',
+      title: t(translations.group),
+      sortable: true,
+      searchable: true,
+      searchProps: {
+        getValue: (datum) => getJointGroupsName(datum.groups),
+      },
+      cell: (datum) => getJointGroupsName(datum.groups),
+      csvDownloadable: true,
+    },
+    {
+      of: 'workflowState',
+      title: t(translations.workflowState),
+      sortable: true,
+      cell: (datum) => (
+        <Chip
+          className="w-100"
+          label={
+            statusTranslations[datum.workflowState ?? workflowStates.Unstarted]
+          }
+          style={{
+            backgroundColor:
+              palette.submissionStatus[
+                datum.workflowState ?? workflowStates.Unstarted
+              ],
+          }}
+          variant="filled"
+        />
+      ),
+      className: 'center',
+    },
+    ...statColumns,
+    {
+      searchProps: {
+        getValue: (datum) =>
+          datum.liveFeedbackCount
+            ? datum.liveFeedbackCount
+                .reduce((sum, count) => sum + (count || 0), 0)
+                .toString()
+            : '',
+      },
+      title: t(translations.total),
+      cell: (datum): ReactNode => {
+        const totalFeedbackCount = datum.liveFeedbackCount
+          ? datum.liveFeedbackCount.reduce(
+              (sum, count) => sum + (count || 0),
+              0,
+            )
+          : null;
+        return (
+          <div className="p-[1rem]">
+            <Box>{totalFeedbackCount}</Box>
+          </div>
+        );
+      },
+      sortable: true,
+      csvDownloadable: true,
+      className: 'text-right',
+      sortProps: {
+        sort: (a, b): number => {
+          const totalA = a.liveFeedbackCount
+            ? a.liveFeedbackCount.reduce((sum, count) => sum + (count || 0), 0)
+            : 0;
+          const totalB = b.liveFeedbackCount
+            ? b.liveFeedbackCount.reduce((sum, count) => sum + (count || 0), 0)
+            : 0;
+          return totalA - totalB;
+        },
+      },
+    },
+  ];
+
+  return (
```

Comment:
Is it possible to refactor this file furthermore into the following:

`LiveHelpStatistics.tsx` -> props is includePhantom
```
<Preload
      render={<LoadingIndicator />}
      while={fetchAndSetLiveFeedbackStatistics}
 >
   {(): JSX.Element => <LiveHelpStatisticsTable includePhantom={includePhantom} />}
</Preload>
```

basically
- you need to rename this file into LiveHelpStatistics.tsx, then
- create new file for LiveHelpStatisticsTable.tsx

This is to create consistencies among the codebase. If you're doing it this way, the `Preload` component is quite obscured and hence it might not be that apparent that the call for `fetchAndSetLiveFeedbackStatistics` needs to finish first before the table itself is rendered

Reply:
done